docs/pages/management/operations/self-hosted-automatic-agent-updates.mdx

```diff
@@ -27,12 +27,9 @@ already done, or you are a Teleport Cloud user, you can directly
 Systemd agents enrolled into automatic updates can only install versions
 present in their package repositories. As Teleport 14 won't be published to
 `stable/v13`, those agents will require manual intervention to be updated to
-the next major version (adding a new apt/yum repo for `stable/v14`).
-
-This limitation will be fixed before Teleport 14. We'll provide a rolling
-update channel for agents to be able to automatically update from one major
-to another. When this will happen, you will have to update the repos of the
-existing agents to point to the new repo.
+the next major version (adding a new APT/YUM/zypper repo for `stable/v14`).
+Alternatively, the `stable/rolling` channel can be used which contains
```

Comment:
```suggestion
Alternatively, you can use the `stable/rolling` channel, which contains
```

Reply:
This changes the text to a second person perspective. Is that intended? I think most of our docs are third person, but may be mistaken.

Reply:
It's pretty common in our docs (and docs in general) to address the reader. I made this change to use active voice instead of passive.